marketplace/resource_wellness_offering.go

```diff
@@ -0,0 +1,84 @@
+package marketplace
+
+import (
+	"errors"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+)
+
+var errNotImplemented = errors.New("resource handler not implemented yet")
+
+func createWellnessOffering(d *schema.ResourceData, meta interface{}) error {
+	return errNotImplemented
+}
+
+func readWellnessOffering(d *schema.ResourceData, meta interface{}) error {
+	return errNotImplemented
+
+}
+
+func updateWellnessOffering(d *schema.ResourceData, meta interface{}) error {
+	return errNotImplemented
+}
+
+func deleteWellnessOffering(d *schema.ResourceData, meta interface{}) error {
+	return errNotImplemented
+}
+
+func wellnessOfferingResource() *schema.Resource {
+	return &schema.Resource{
+		Schema: map[string]*schema.Schema{
+			"title": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"description": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"marketplace_provider": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"version": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"auto_version": {
+				Type:     schema.TypeBool,
+				Optional: true,
+			},
+			"imageUrl": {
```

Comment:
Non-blocking nit: `imageUrl` should probably be `image_url` to stick with attribute naming conventions in Terraform
```suggestion
			"image_url": {
```